vmm/src/memory_manager.rs

```diff
@@ -450,10 +500,32 @@ impl MemoryManager {
         Ok(slot)
     }
 
+    pub fn virtiomem_resize(&mut self, size: u64) -> Result<(), Error> {
```

Comment:
Can you unify this with `resize()` if you follow the other feedback i've given.

Reply:
Fixed.